terraform/development/main.tf

```diff
@@ -0,0 +1,28 @@
+# INSTRUCTIONS: 
```

Comment:
I wonder if some of the instructional comments here can be removed, assuming they now refer to a completed step?

Some of these comments are perhaps more informational/relevant to future developers opening this file (e.g. line 7?) so may be important to keep.

Reply:
Yeah makes sense, the repo does link back to the template which has the instructions in as well.